comtypes/__init__.py

```diff
@@ -302,10 +330,10 @@ class _cominterface_meta(type):
     _com_shutting_down = False
 
     # Creates also a POINTER type for the newly created class.
-    def __new__(cls, name, bases, namespace):
+    def __new__(mcs, name, bases, namespace):
```

Comment:
From the pep8 docs.

https://peps.python.org/pep-0008/#function-and-method-arguments

> Always use self for the first argument to instance methods.
> 
> Always use cls for the first argument to class methods.
> 
> If a function argument’s name clashes with a reserved keyword, it is generally better to append a single trailing underscore rather than use an abbreviation or spelling corruption. Thus class_ is better than clss. (Perhaps better is to avoid such clashes by using a synonym.)

Now it defines cls as being only used in "class methods" which `__new__` is not. `__new__` is also not an instance method either. so mcs is the appropriate  first parameter name. Flake8 was changed without reviewing the pep documentation. I am going to open an issue for it on flake8 repo to have it corrected. I believe where the issue was when it was requiring  that mcs be used outside of a class method and they ended up changing it for all class types instead of enforcing the use of mcs for metaclasses only.